router/router.go

```diff
@@ -663,10 +663,28 @@ func (worker *workerT) processDestinationJobs() {
 								if worker.rt.transformerProxy {
 									jobID := destinationJob.JobMetadataArray[0].JobID
 									pkgLogger.Debugf(`[TransformerProxy] (Dest-%[1]v) {Job - %[2]v} Request started`, worker.rt.destName, jobID)
+
+									// setting metadata
+									firstJobMetadata := destinationJob.JobMetadataArray[0]
+									proxyReqMetadata := transformer.ProxyRequestMetadata{
+										SourceID:      firstJobMetadata.SourceID,
+										DestinationID: firstJobMetadata.DestinationID,
+										WorkspaceID:   firstJobMetadata.WorkspaceID,
+										JobID:         firstJobMetadata.JobID,
+										AttemptNum:    firstJobMetadata.AttemptNum,
+										DestInfo:      firstJobMetadata.DestInfo,
+										Secret:        firstJobMetadata.Secret,
+									}
+									// forming the proxy request payload
+									proxyReqPayload := transformer.ProxyRequestPayload{
+										PostParametersT: &result[resultInd],
+										Metadata:        proxyReqMetadata,
+									}
+									// forming proxy request method parameters
 									proxyReqparams := &transformer.ProxyRequestParams{
 										DestName:     worker.rt.destName,
 										JobID:        jobID,
-										ResponseData: val,
+										ResponseData: proxyReqPayload,
 									}
```

Comment:
we could simplify using less variables/comments

```go
									firstJobMetadata := destinationJob.JobMetadataArray[0]
jobID := firstJobMetadata.JobID
pkgLogger.Debugf(`[TransformerProxy] (Dest-%[1]v) {Job - %[2]v} Request started`, worker.rt.destName, jobID)

proxyReqparams := &transformer.ProxyRequestParams{
	DestName: worker.rt.destName,
	JobID:    jobID,
	ResponseData: transformer.ProxyRequestPayload{
		PostParametersT: &result[resultInd],
		Metadata: transformer.ProxyRequestMetadata{
			SourceID:      firstJobMetadata.SourceID,
			DestinationID: firstJobMetadata.DestinationID,
			WorkspaceID:   firstJobMetadata.WorkspaceID,
			JobID:         firstJobMetadata.JobID,
			AttemptNum:    firstJobMetadata.AttemptNum,
			DestInfo:      firstJobMetadata.DestInfo,
			Secret:        firstJobMetadata.Secret,
		},
	},
}

```

Reply:
Done